client/components/focusable/docs/example.jsx

```diff
@@ -0,0 +1,66 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import Gridicon from 'gridicons';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import config from 'config';
+import DocsExample from 'devdocs/docs-example';
+import Focusable from 'components/focusable';
+import ScreenReaderText from 'components/screen-reader-text';
+
+class FocusableExample extends React.PureComponent {
+	static displayName = 'Focusable';
+
+	render() {
+		return config.isEnabled( 'devdocs/components-usage-stats' )
+			? this.renderDocsExampleWithUsageStats()
+			: this.renderFocusable();
+	}
+
+	renderDocsExampleWithUsageStats = () => {
+		return (
+			<DocsExample componentUsageStats={ this.props.componentUsageStats }>
+				{ this.renderFocusable() }
+			</DocsExample>
+		);
+	};
+
+	renderFocusable = () => {
+		return (
+			<Card>
+				<Focusable onClick={ noop }>
+					<p>
+						This keyboard-accessible component can contain other elements as children, including{' '}
```

Comment:
I'd add a warning here too, and link back to the readme 👍

Reply:
Oh I forgot that we show the readme now on http://calypso.localhost:3000/devdocs/design/focusable ! Could we maybe make the warning in the readme have some extra styling to make it stand out?